Studio/Linux.md

```diff
@@ -1,158 +1,122 @@
+The following .rpm packages need to be installed:
 
-                   *cups-devel libXtst-devel alsa-lib-devel libXrandr-devel \\*
+```bash
+    yum install at-spi2-atk cups-libs libdrm libgbm gtk3 alsa-lib libsecret
+```
 
-                   *nss-devel libsecret-devel libnotify-devel wget unzip*
+#### On Ubuntu / Debian (and other Deb packaging based Linux distributions)
 
-&nbsp;
+The following .deb packages need to be installed
+```bash
+    apt-get install \
+        libgbm1 \
+		libglib2.0-0 \
+		libgtk-3-0 \
+		libkrb5-3 \
+		libnss3 \
+		libnss-wrapper \
+		libsecret-1-0 \
+		libssl-dev
+```
 
-Copy and replace [file](<https://drive.google.com/file/d/17aJr9wHbl\_xPtCZpyM7xygiObC7lzHTB/view?usp=sharing> "target=\"\_blank\"") in the directory ([link](<https://drive.google.com/file/d/17aJr9wHbl\_xPtCZpyM7xygiObC7lzHTB/view?usp=sharing> "target=\"\_blank\"")):
+##### On servers
 
-*/lib64/libstdc++.so.6*
+If you want to run Hackolade only in cli on a server without a GUI environment installed you'll still require some more tooling:
 
-[https://drive.google.com/file/d/17aJr9wHbl\_xPtCZpyM7xygiObC7lzHTB/view?usp=sharing](<https://drive.google.com/file/d/17aJr9wHbl\_xPtCZpyM7xygiObC7lzHTB/view?usp=sharing> "target=\"\_blank\"")
-
-&nbsp;
-
-*sudo yum groupinstall “GNOME Desktop”*
-
-*echo "exec gnome-session" \>\> ~/.xinitrc*
-
-&nbsp;
-
-Starting GNOME:
-
-*startx*
-
-&nbsp;
-
-*Notice*: you should start GNOME by a non-root user
-
-[https://www.techrepublic.com/article/how-to-install-a-gui-on-top-of-centos-7/](<https://www.techrepublic.com/article/how-to-install-a-gui-on-top-of-centos-7/> "target=\"\_blank\"")
+```bash
+   yum install Xvfb
+```
+or
+```bash
+   apt-get install xvfb
+```
 
 ### Installation
 
-*wget “[https://s3-eu-west-1.amazonaws.com/hackolade/current/Hackolade-linux-x64.zip*](<https://s3-eu-west-1.amazonaws.com/hackolade/current/Hackolade-linux-x64.zip> "target=\"\_blank\"")*” -O hackolade.zip*
-
-*unzip hackolade.zip*
-
-*rm hackolade.zip*
-
-&nbsp;
-
-*cd Hackolade-linux-x64 *
-
-*chown root:root chrome-sandbox*
-
-*chmod 4755 chrome-sandbox*
+Download the [latest release](<https://s3-eu-west-1.amazonaws.com/hackolade/current/Hackolade-linux-x64.zip> "target=\"\_blank\"")
+```bash
+    wget https://s3-eu-west-1.amazonaws.com/hackolade/current/Hackolade-linux-x64.zip
+```
 
+Then unzip:
+```bash
+    unzip hackolade.zip
+    rm hackolade.zip
+    cd Hackolade-linux-x64
+```
 &nbsp;
 
 *Notice*:
-
 File “chrome-sandbox” must be owned by root user and have permissions 4755.
 
+```bash
+    sudo chown root:root chrome-sandbox
+    sudo chmod 4755 chrome-sandbox
+```
+
 &nbsp;
 
 The application must be executed by a non-root user.
 
 ### Start application
 
-Execute in the terminal:
-
-*./Hackolade*
-
-### Remote connection
-
-In case of remote connection, you need VNC server:
-
-[https://wiki.centos.org/HowTos/VNC-Server](<https://wiki.centos.org/HowTos/VNC-Server> "target=\"\_blank\"")
-
-### Compiling GCC
```

Comment:
GCC doesn't really have to do anything with Hackolade. It is actually a C compiler... The possible real issue is with the GLIBC standard library that might be outdated but in such a case telling the user to compile a newer version of the GLIBC might create more issues than the ones it possible solves.